src/Public/Routes.ps1

```diff
@@ -68,6 +68,17 @@ One or more optional Scopes that will be authorised to access this Route, when u
 .PARAMETER User
 One or more optional Users that will be authorised to access this Route, when using Authentication with an Access method.
 
+.PARAMETER OAResponses
+An alternative way to associate OpenApi responses unsing New-PodeOAResponse instead of piping multiple Add-PodeOAResponse
+
+.PARAMETER OAReference
+A reference to OpenAPI reusable pathItem component created with Add-PodeOAComponentPathItem
+
+.PARAMETER DefinitionTag
```

Comment:
I'm thinking for this parameter, similar to `OAReference` and `OAResponses` above, would also benefit from being called `OADefinitionTag`. Just so it's obvious it's related to OpenAPI.

Reply:
Changed